### PR TITLE
ref(ts): Convert InactivePlugins to typescript

### PR DESCRIPTION
**Pull request by matejminar**

### CODE REVIEW

File: src/sentry/static/sentry/app/components/inactivePlugins.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  width: 175px;
+  text-align: center;
+  font-size: ${p => p.theme.fontSizeSmall};
+  color: #889ab0;
+  letter-spacing: 0.1px;
+  font-weight: 600;
+  text-transform: uppercase;
+  border: 1px solid #eee;
+  background: inherit;
+  border-radius: 4px;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/inactivePlugins.tsx:63]:
```suggestion
  border-radius: ${p => p.theme.borderRadius};
```

File: src/sentry/static/sentry/app/components/inactivePlugins.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  padding: 10px;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/inactivePlugins.tsx:64]:
```suggestion
  padding: ${space(1)};
```

File: src/sentry/static/sentry/app/components/inactivePlugins.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+  &:hover {
+    border-color: #ccc;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/inactivePlugins.tsx:67]: maybe you can replace these colors with colors of our theme?

**Review by priscilawebdev — COMMENTED** [review]
It looks great..I left just a few suggestions 😉

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/inactivePlugins.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/inactivePlugins.tsx]: Thanks!

File: src/sentry/static/sentry/app/components/inactivePlugins.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/inactivePlugins.tsx:64]: These values are not the same and I would rather not change the design as part of this typescript conversion PR.

**Pull request merged**